worldengine/common.py

```diff
@@ -120,3 +120,33 @@ def array_to_matrix(array, width, height):
         for x in range(width):
             matrix[y].append(array[y * width + x])
     return matrix
+
+def _equal(a, b):
```

Comment:
I hope this doesn't seem too much like a hack. I just didn't know how to handle numpy.array== otherwise (it defaults to returning an array of bools instead of a single bool, which did cause problems at many places in the code).